pkg/v1/tkg/test/tkgctl/docker_cc/docker_suite_test.go

```diff
@@ -0,0 +1,214 @@
+			Timeout:                     timeout,
+			Size:                        e2eConfig.ManagementClusterOptions.Size,
+			DeployTKGonVsphere7:         e2eConfig.ManagementClusterOptions.DeployTKGonVsphere7,
+			EnableTKGSOnVsphere7:        e2eConfig.ManagementClusterOptions.EnableTKGSOnVsphere7,
+			VsphereControlPlaneEndpoint: e2eConfig.ManagementClusterOptions.Endpoint,
+			CniType:                     "antrea",
+			Edition:                     "tkg",
+		})
+
+		Expect(err).To(BeNil())
+	}
+
+	//// Create initial workload cluster
+	//clusterName = e2eConfig.ClusterPrefix + "wc"
+	//options := framework.CreateClusterOptions{
+	//	ClusterName: clusterName,
+	//	Namespace:   constants.DefaultNamespace,
+	//	Plan:        "dev",
+	//}
+	//clusterConfigFile, err := framework.GetTempClusterConfigFile(e2eConfig.TkgClusterConfigPath, &options)
+	//Expect(err).To(BeNil())
+	//
+	//defer os.Remove(clusterConfigFile)
+	//
+	//tkgCtlClient, err := tkgctl.New(tkgctl.Options{
+	//	ConfigDir: e2eConfig.TkgConfigDir,
+	//	LogOptions: tkgctl.LoggingOptions{
+	//		File:      filepath.Join(logsDir, clusterName+".log"),
+	//		Verbosity: e2eConfig.TkgCliLogLevel,
+	//	},
+	//})
+	//Expect(err).ToNot(HaveOccurred())
+	//
+	//optOutStatus, err := tkgCtlClient.GetCEIP()
+	//Expect(err).ToNot(HaveOccurred())
+	//Expect(optOutStatus.CeipStatus).To(Equal("Opt-out"))
+	//
+	//By(fmt.Sprintf("Creating initial workload cluster %q", clusterName))
+	//
+	//defer os.Remove(clusterConfigFile)
+	//err = tkgCtlClient.CreateCluster(tkgctl.CreateClusterOptions{
+	//	ClusterConfigFile: clusterConfigFile,
+	//	Edition:           "tkg",
+	//})
+	//Expect(err).To(BeNil())
+
+	return []byte(
+		strings.Join([]string{
+			artifactsFolder,
+			e2eConfigPath,
+		}, ","),
+	)
+}, func(data []byte) {
+	// Before each parallel node
+
+	parts := strings.Split(string(data), ",")
+	Expect(parts).To(HaveLen(2))
+
+	artifactsFolder = parts[0]
+	e2eConfigPath = parts[1]
+	e2eConfig = framework.LoadE2EConfig(context.TODO(), framework.E2EConfigInput{ConfigPath: e2eConfigPath})
+	Expect(e2eConfigPath).ToNot(BeNil(), "Failed to load e2e config from %s", e2eConfigPath)
+})
+
+var _ = SynchronizedAfterSuite(func() {
+}, func() {
+	// After all parallel nodes
+
+	timeout, err := time.ParseDuration(e2eConfig.DefaultTimeout)
+	Expect(err).To(BeNil())
+
+	//By(fmt.Sprintf("Deleting workload cluster %q", clusterName))
+
+	//logsDir := filepath.Join(artifactsFolder, "logs")
+	//tkgCtlClient, err := tkgctl.New(tkgctl.Options{
+	//	ConfigDir: e2eConfig.TkgConfigDir,
+	//	LogOptions: tkgctl.LoggingOptions{
+	//		File:      filepath.Join(logsDir, clusterName+".log"),
+	//		Verbosity: e2eConfig.TkgCliLogLevel,
+	//	},
+	//})
+	//Expect(err).ToNot(HaveOccurred())
+	//
+	//err = tkgCtlClient.DeleteCluster(tkgctl.DeleteClustersOptions{
+	//	ClusterName: clusterName,
+	//	Namespace:   constants.DefaultNamespace,
+	//	SkipPrompt:  true,
+	//})
+	//Expect(err).To(BeNil())
```

Comment:
Is there a reason to keep this as commented or should we remove this?

Reply:
Good catch. Deleted